src/components/maincontroller.js

```diff
@@ -36,21 +35,13 @@ window.mediaManager = window.castReceiverContext.getPlayerManager();
 
 const playbackMgr = new playbackManager(window.mediaManager);
 
-var init = function () {
-    resetPlaybackScope($scope);
-};
-
-init();
+resetPlaybackScope($scope);
 
 var mgr = window.mediaManager;
 
 var broadcastToServer = new Date();
 
 export function onMediaElementTimeUpdate() {
-    if ($scope.isChangingStream) {
-        return;
-    }
```

Comment:
these are actually needed to inhibit the event handlers from screwing up stuff when we are e.g. continuing in the playlist or server seeking/changing a stream. By removing them I have broken stuff while we really should make sure the flag is set somewhere where relevant...